packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -259,14 +325,22 @@ export const AddNewTrustline: FC = () => {
           </>
         }
         transaction={TransactionStatus.Rejected}
+        {...(params.inAppCall ? { onClick: () => navigate(HOME_PATH) } : {})}
       />
     );
   }
 
   if (!isValidIssuer) {
```

Comment:
Some comment than above:
```suggestion
  if (!isValidIssuer && !params.inAppCall) {
```

Reply:
Not exactly, we still want to return <AsyncTransaction ... if isValidIssuer, even if it's not an inApp call